contentcuration/contentcuration/management/commands/setup_perftest_data.py

```diff
@@ -17,22 +17,27 @@ def handle(self, *args, **options):
 
         self.editor = create_user("ivanbot@leq.org", "ivanisthe1", "Ivan", "NeoBot")
 
+        self.user_swarm = [self.editor]
+        for i in range(100):
+            self.user_swarm.append(create_user("ivanbot{}@leq.org".format(i), "ivanisthe1", "Ivan", "NeoBot"))
```

Comment:
This produces an error if the user already exists. In order to do the script idempotent, please use `get_or_create` instead